sdk/python/feast/feature_view.py

```diff
@@ -176,3 +177,65 @@ def most_recent_end_time(self) -> Optional[datetime]:
         if len(self.materialization_intervals) == 0:
             return None
         return max([interval[1] for interval in self.materialization_intervals])
+
+    def run_reverse_field_mapping(
+        self, join_keys: List[str],
```

Comment:
The name `run_reverse_field_mapping` provides almost no intent to the user. Would `get_field_names()` be better?

Reply:
changed run_reverse_field_mapping to _get_column_names; I think returning a tuple vs a dictionary are both fine, and inclined to just leave as is until we have a reason to change things. BTW this method is not user-facing and now starts with an underscore